dpx/src/dpx_dpxutil.rs

```diff
@@ -87,8 +86,7 @@ pub unsafe extern "C" fn min4(mut x1: f64, mut x2: f64, mut x3: f64, mut x4: f64
     }
     v
 }
```

Comment:
`x1.min(x2).min(x3).min(x4)` is more simple, so as `max4`.

Reply:
Sorry, I pressed the review button incautiously

Reply:
I've already removed `min4`/`max4` in #148 .